README.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+output: github_document
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# Calculate the final size of an epidemic <img src="man/figures/logo.png" align="right" width="120"/>
+
+<!-- badges: start -->
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![R-CMD-check](https://github.com/epiverse-trace/finalsize/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/epiverse-trace/finalsize/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/epiverse-trace/finalsize/branch/main/graph/badge.svg)](https://app.codecov.io/gh/epiverse-trace/finalsize?branch=main)
+<!-- badges: end -->
+
+_finalsize_ provides quick back-of-the-envelope calculations for the final size of an epidemic outbreak in a population with demographic variation in contact patterns, and variation within and between age groups in their susceptibility to disease.
+
+_finalsize_ can help provide rough estimates of the effectiveness of pharmaceutical interventions in the form of immunisation programmes, or the effect of naturally acquired immunity through previous infection (see the vignette).
+
+_finalsize_ relies on [Eigen](https://eigen.tuxfamily.org/) via [RcppEigen](https://github.com/RcppCore/RcppEigen) for fast matrix algebra, and is developed at the [Centre for the Mathematical Modelling of Infectious Diseases](https://www.lshtm.ac.uk/research/centres/centre-mathematical-modelling-infectious-diseases) at the London School of Hygiene and Tropical Medicine as part of the [Epiverse Initiative](https://data.org/initiatives/epiverse/).
+
+## Installation
+
+You can install the development version of finalsize from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("epiverse-trace/finalsize")
+
+# load finalsize
+library(finalsize)
+```
+
+## Quick start
+
+_finalsize_ provides the single function `final_size()`, to calculate the final size of an epidemic.
```

Comment:
Italicised package name here, and `monospaced` for socialmixr, just to bring to your attention in case you want package names formatted consistently.

Reply:
This is something for which I haven't found a good solution: perhaps we should use `{...}` for package names? Not monospaced.

Reply:
Changed to italics for package names.